src/app/views/orders/order.html

```diff
@@ -2,6 +2,39 @@
   <a ui-sref="dashboard.orders" class="text-muted">
     <i class="fa fa-chevron-left"></i>&nbsp;<span translate>mnoe_admin_panel.dashboard.order.back</span>
   </a>
+  <div class="row">
+    <div class="col-xs-12">
+      <mno-widget icon="fa-user-o" heading="{{'mnoe_admin_panel.dashboard.order.customer' | translate}}" is-loading="vm.isLoading">
+        <mno-widget-header></mno-widget-header>
+        <mno-widget-body>
+          <div class="row">
```

Comment:
For these rows, any reason the columns add up to 10? Should usually take up the entire 12.

Reply:
@adamaziz15 I was just moving original code up from below. I can fix this and go 4 and 8.

Reply:
@adamaziz15 changed